components/GUI/xp_controller_widget.py

```diff
@@ -55,7 +55,7 @@ def get_completion_value(priority : priority_t, projects : list[str] | None, tag
         int
             The computed completion value after applying all relevant multipliers.
         """
-        # completion_value : int = XpControllerWidget.PRIORITY_MULT_MAP[priority if priority != None else '']  # Get the base completion value
+
         completion_value = XpControllerWidget.PRIORITY_MULT_MAP.get(priority, 0.5)  # fixes no priority task error
```

Comment:
Right now, it always defaults to 0.5 if there is no priority. But, we want the user to be able to define that, which is why you added it to the config dictionary above. However, that value is never gotten because 'None' is not a valid value for priority_t, so this fixes it.

I did not check if this was 100% valid code so please do.

```suggestion
        completion_value = XpControllerWidget.PRIORITY_MULT_MAP.get(priority, PRIORITY_MULT_MAP['None'])  # fixes no priority task error
```